f/medical/f_clsEH.sqf

```diff
@@ -0,0 +1,55 @@
+// FA3 - Combat Life Saver Event Handler
+// This component adds an event handler for use with the Combat Life Saver assignGear class. Units with the f3_cls trait can provide full heals using FAKs.
+// Enable this component in init.sqf
+// For credits and more information see https://github.com/folkarps/F3/wiki
+
+// Make sure the player is initialised
+if (!isDedicated && (isNull player)) then
+{
+	waitUntil {sleep 0.1; !isNull player};
+};
+
+// Make sure the player can be broadcast properly
+_unit = player;
+
+// Add the EH
+[_unit, ["HandleHeal", {
+	_this spawn {
+		params ["_injured", "_healer","_isMedic"];
+			
+
+		
+			// Check whether the person healing the player is a CLS
+			if (_healer getUnitTrait "f3_cls") then {
+			
+				// DEBUG
+				if (f_param_debugMode == 1) then
+					{
+						player sideChat format ["DEBUG (f_clsEH.sqf): %1 is CLS healing %2",(name _healer),(name _injured)];
+					};
+					
+				// Wait until the standard heal has been applied, or a timeout happens
+				_timeout = (time + 20);
+				waitUntil {(damage _injured <= 0.2501) or (time > _timeout)};
+
+				// If it timed out, exit with nothing but a debug message
+				if (time > _timeout) then {
+					exitWith {
```

Comment:
This syntax is wrong, it should be `if () exitwith {}`. Also I think you're missing a bracket? I don't think this code would actually work.

Reply:
Fixed

Reply:
Still not quite right. `exitWith` basically replaces `then`, you don't need a bracket between it and the if statement.

Reply:
Look, in fairness, reading is an optional skill here at FA